test/components/upgrades.test.js

```diff
@@ -12,23 +12,25 @@ describe('Upgrades testing', function () {
     describe('Agent Registry', async function () {
         it(' 0.1.1 -> 0.1.5', async function () {
             const AgentRegistry_0_1_1 = await ethers.getContractFactory('AgentRegistry_0_1_1');
-            agents = await upgrades.deployProxy(AgentRegistry_0_1_1, [this.contracts.access.address, this.contracts.router.address, 'Forta Agents', 'FAgents'], {
+            const MOCK_ROUTER = this.contracts.access.address;
+            agents = await upgrades.deployProxy(AgentRegistry_0_1_1, [this.contracts.access.address, MOCK_ROUTER, 'Forta Agents', 'FAgents'], {
                 constructorArgs: [this.contracts.forwarder.address],
                 unsafeAllow: ['delegatecall'],
             });
             await agents.deployed();
-
+            console.log(agents)
             //create agent
             const AGENT_ID = ethers.utils.hexlify(ethers.utils.randomBytes(32));
+            console.log(AGENT_ID);
             const args = [AGENT_ID, this.accounts.user1.address, 'Metadata1', [1, 3, 4, 5]];
             await agents.prepareAgent(prepareCommit(...args));
             await network.provider.send('evm_increaseTime', [300]);
-            await expect(agents.connect(this.accounts.other).createAgent(...args));
+            await agents.connect(this.accounts.user1).createAgent(...args);
 
             // Checks
-            //expect(await this.agents.isCreated(AGENT_ID)).to.be.equal(true); //Does not exist in 0.1.1
             expect(await agents.name()).to.be.equal('Forta Agents');
             expect(await agents.symbol()).to.be.equal('FAgents');
+            console.log('lol');
```

Comment:
A few leftover console.logs in this file. :stuck_out_tongue_winking_eye:

Reply:
```
console.log(😅)
```
Are you using a script or tool to catch those?

Reply:
No just reviewing the code!

Reply:
Removed in 6a30940